source/src/processing/cs_PowerSampling.cpp

```diff
@@ -352,13 +357,18 @@ void PowerSampling::powerSampleAdcDone(adc_buffer_id_t bufIndex) {
 
 	if (!isConsecutiveBuf(seqNr, _lastBufSeqNr)) {
 		LOGw("buf skipped (prev=%u cur=%u)", _lastBufSeqNr, seqNr);
+
+		adc_buffer_seq_nr_t skips = seqNr - 1 - _lastBufSeqNr;
```

Comment:
the adc buffer is circular, right? is this computation correct?

Reply:
pretty sure i checked this for `isConsecutiveBuf`